pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -60,6 +61,9 @@
                                     <%= question.title %></a>
                                 <%- include('../partials/issueBadge', {count: question.open_issue_count, issueQid: question.qid}); %>
                             </td>
+                            <td class="align-middle">
+                                <image src="<%= urlPrefix %>/course_admin/questionThumbnail/<%= question.qid %>" onerror="this.style.display='none'" width="100" height="100"/>
```

Comment:
We generally use numeric IDs in URLs to ensure we get exactly the object we want. Here that would mean using `question.id` rather than `question.qid`.

Reply:
The question.qid is conveniently the name of the file the image is stored in. Is there a way to go between the two? So that I can pass in the question.id into the get request and still find the folder.